### PR TITLE
(PDFCreator) Better installs for non-domain systems and AHK script fixes

### PR DESCRIPTION
**Pull request by teknowledgist**
## Description
This makes non-domain systems silently install "normally" (i.e. with a `/VERYSILENT` option).  Domain joined systems must use the (improved) AHK scripted method.  

Also added some information in the description for install arguments (`--ia`) that are likely to be commonly sought. 

## Motivation and Context
This should resolve [this issue ](https://github.com/chocolatey/chocolatey-coreteampackages/issues/1144) and make the package more standard and better behaved for non-domain-joined systems.

## How Has this Been Tested?
Package tested on a non-domain-joined, Win7 system and a domain-joined Win10 system.  

There are a few inconsistencies between the two methods:

* Domain systems do not get a desktop icon.  Non-domain systems do get a desktop icon unless they opt for no icons at all (with the `--ia '"/NOICONS"'` switch).
* Domain systems sometimes get one or more browser windows opening to advertise PDF Architect.
* The `--ia '"/TASKS=!winexplorer"'` option noted in the description does not always take effect.

## Types of changes
- [x ] Bug fix (non-breaking change which fixes an issue)
- [? ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Migrated package (a package has been migrated from another repository)

## Checklist:

- [ ] My code follows the code style of this repository.
- [ ] My change requires a change to documentation (this usually means the notes in the description of a package).
- [x ] I have updated the documentation accordingly (this usually means the notes in the description of a package).
- [ ] All files are up to date with the latest [Contributing Guidelines](https://github.com/chocolatey/chocolatey-coreteampackages/blob/master/CONTRIBUTING.md)
- [ ] The added/modified package passed install/uninstall in the chocolatey test environment.
    *I have never been able to get the test environment up and running.*
- [x ] The changes only affect a single package (not including meta package).

### CODE REVIEW

File: automatic/pdfcreator/pdfcreator.nuspec [hunk 1]
```diff
@@ -23,10 +23,27 @@ PDFCreator easily creates PDFs from any Windows program. Use it like a printer i
...
 - Profiles make frequently used settings available with one click
 - Use automatic saving to have a fully automated PDF printer
+- Secure against opening, printing or modifying your PDFs with a password
+- Support for digital signatures
+- COM interface for application or scripted actions
+- PDFCreator is Open Source software and licensed under the terms of the Affero General Public License (AGPL)
+- You are free to use PDFCreator at home or at work
+- You may access the source code and compile it on your own, as long as it stays under the AGPL
+
+### Install Options
```
Comment (majkinetor) [inline automatic/pdfcreator/pdfcreator.nuspec]: Please change this to `## Package Parameters`

File: automatic/pdfcreator/tools/chocolateyInstall.ps1 [hunk 2]
```diff
@@ -36,22 +36,30 @@ try {
...
   Write-Warning "Unexpected error while checking Print Spooler service: $($_.Exception.Message)"
 }
 
-# silent install requires AutoHotKey
-$ahkFile = Join-Path $toolsPath 'chocolateyInstall.ahk'
-$ahkEXE = Get-ChildItem "$env:ChocolateyInstall\lib\autohotkey.portable" -Recurse -filter autohotkey.exe
-$ahkProc = Start-Process -FilePath $ahkEXE.FullName -ArgumentList "$ahkFile" -PassThru
-Write-Debug "AutoHotKey start time:`t$($ahkProc.StartTime.ToShortTimeString())"
-Write-Debug "Process ID:`t$($ahkProc.Id)"
+# silent install requires different methods for domain-joined systems
```
Comment (majkinetor) [inline automatic/pdfcreator/tools/chocolateyInstall.ps1]: Too much details in main installer. Please make it use `helper.ps1` with adequately named functions. See [1.1.7](https://github.com/chocolatey/chocolatey-coreteampackages/blob/master/CONTRIBUTING.md#117-clean-code)

Comment (teknowledgist) [reply automatic/pdfcreator/tools/chocolateyInstall.ps1]: All of that AHK "stuff" is in the current package, I just made its use dependent on domain-joined-status.  It's interesting that including it was acceptable the last time.

I'm not sure how I would separate that into separate functions.  Are there other packages that use AHK that do it "properly"?  

If using AHK is too detailed for the main installer and a common "issue", someone should make a Chocolatey extension package to handle in a standardized way.

File: automatic/pdfcreator/pdfcreator.nuspec (hunk 1 above)
Comment (majkinetor) [reply automatic/pdfcreator/pdfcreator.nuspec]: See [1.2.3](https://github.com/chocolatey/chocolatey-coreteampackages/blob/master/CONTRIBUTING.md#123-informative-description) please for details.

File: automatic/pdfcreator/tools/chocolateyInstall.ps1 (hunk 2 above)
Comment (majkinetor) [reply automatic/pdfcreator/tools/chocolateyInstall.ps1]: I am not talking about AHK,  but about `chocolateyInstall.ps1`. Yeah, there are number of packages, just hit Find file here and look for `helpers.ps1` files.

Comment (teknowledgist) [reply automatic/pdfcreator/tools/chocolateyInstall.ps1]: I'm sorry, but I not sure I understand.  What detail needs to be moved to a function?  The AHK process creation is the same as before.  I added an "if part of domain" and a couple messages for the user.

File: automatic/pdfcreator/pdfcreator.nuspec (hunk 1 above)
Comment (teknowledgist) [reply automatic/pdfcreator/pdfcreator.nuspec]: OK, so not '## Package Parameters`, but `## Notes`.

File: automatic/pdfcreator/tools/chocolateyInstall.ps1 (hunk 2 above)
Comment (teknowledgist) [reply automatic/pdfcreator/tools/chocolateyInstall.ps1]: I added an intermediate step to make it more readable.

**Pull request merged**